mteb/tasks/Retrieval/multilingual/XPQARetrieval.py

```diff
@@ -74,33 +71,107 @@ class XPQARetrieval(MultilingualTask, AbsTaskRetrieval):
         },
         type="Retrieval",
         category="s2p",
-        eval_splits=[_EVAL_SPLIT],
-        eval_langs=_LANGS,
+        eval_splits=["test"],
+        eval_langs=_EVAL_LANGS,
         main_score="ndcg_at_10",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
-        n_samples=None,
-        avg_character_length=None,
+        date=("2022-01-01", "2023-07-31"),  # best guess
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Question answering"],
+        license="CDLA-Sharing-1.0",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""@inproceedings{shen2023xpqa,
+        title={xPQA: Cross-Lingual Product Question Answering in 12 Languages},
+        author={Shen, Xiaoyu and Asai, Akari and Byrne, Bill and De Gispert, Adria},
+        booktitle={Proceedings of the 61st Annual Meeting of the Association for Computational Linguistics (Volume 5: Industry Track)},
+        pages={103--115},
+        year={2023}
+        }""",
+        n_samples={"test": 19801},
+        avg_character_length={"test": 104.68},  # answer
     )
 
     def load_data(self, **kwargs):
         if self.data_loaded:
             return
 
-        self.corpus, self.queries, self.relevant_docs = _load_xpqa_data(
-            path=self.metadata_dict["dataset"]["path"],
-            langs=self.hf_subsets,
-            split=self.metadata_dict["eval_splits"][0],
-            cache_dir=kwargs.get("cache_dir", None),
-            revision=self.metadata_dict["dataset"]["revision"],
-        )
+        path = self.metadata_dict["dataset"]["path"]
+        revision = self.metadata_dict["dataset"]["revision"]
+        eval_splits = self.metadata_dict["eval_splits"]
+        dataset = _load_dataset_csv(path, revision, eval_splits)
+
+        self.queries, self.corpus, self.relevant_docs = {}, {}, {}
+        for lang_pair, _ in self.metadata.eval_langs.items():
+            lang_corpus, lang_question = (
+                lang_pair.split("-")[0],
+                lang_pair.split("-")[1],
+            )
+            lang_not_english = lang_corpus if lang_corpus != "eng" else lang_question
+            dataset_language = dataset.filter(
+                lambda x: x["lang"] == _LANG_CONVERSION.get(lang_not_english)
+            )
+            question_key = "question_en" if lang_question == "eng" else "question"
+            corpus_key = "candidate" if lang_corpus == "eng" else "answer"
```

Comment:
so candidate and answer are quivalent for all languages?

Reply:
yes more or less, with candidate being in english and answer in the other language